__tests__/app.test.js

```diff
@@ -197,3 +198,68 @@ describe('POST/api/articles/:article_id/comments', () => {
 			});
 	});
 });
+
+describe('PATCH/api/articles/:article', () => {
+	test('responds with a 200 status and the article object with updated vote count', () => {
+		return request(app)
+			.put('/api/articles/1')
+			.send({ inc_votes: 10 })
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toEqual(
+					expect.objectContaining({
+						article_id: expect.any(Number),
+						title: expect.any(String),
+						topic: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						created_at: expect.any(String),
+						votes: 110,
+					})
+				);
+			});
+	});
+	test('returns a 400 status error and error message if no inc_votes object is sent with the request', () => {
+		return request(app)
+			.put('/api/articles/1')
+			.send()
+			.expect(400)
+			.then(({ body }) => {
+				expect(body).toBeInstanceOf(Object);
+				expect(body).toEqual(
+					expect.objectContaining({
+						msg: 'Please provide a votes value',
+					})
+				);
+			});
+	});
+
+	test('returns a 400 status error and error message if inc_votes is equal to 0', () => {
+		return request(app)
+			.put('/api/articles/1')
+			.send({ inc_votes: 0 })
+			.expect(400)
+			.then(({ body }) => {
+				expect(body).toBeInstanceOf(Object);
+				expect(body).toEqual(
+					expect.objectContaining({
+						msg: 'Please provide a votes value',
```

Comment:
_technically_ they have provided a votes value, so this message could be a little misleading in this case.